doc/.gitignore

```diff
@@ -1,4 +1,3 @@
 _site
 .sass-cache
```

Comment:
Is the plan to get ride of this secondary .gitignore file and just keep that one in root?

Reply:
No, this whole directory ends up getting copied over to the https://github.com/phac-nml/irida-documentation repo when we do a release.  The issue was that since the apidocs directory was within the gitignore here, it wasn't updating the apidocs when we released the new doc code.  We want to keep some things ignored (like the `_site` stuff).

Reply:
Sounds good. This looks great then thanks